src/main/java/frc/team2412/robot/Subsystems.java

```diff
@@ -16,8 +16,8 @@
 
 public class Subsystems implements Loggable {
     public static class SubsystemConstants {
-        public static final boolean CLIMB_ENABLED = true;
-        public static final boolean DRIVE_ENABLED = true;
+        public static final boolean CLIMB_ENABLED = false;
+        public static final boolean DRIVE_ENABLED = false;
```

Comment:
We should remove these changes to prevent collisions as we do our final testing and tuning before this weekend's competition.

Reply:
Ok, will update